fluid/face_detection/reader.py

```diff
@@ -278,8 +250,8 @@ def train(settings, file_list, shuffle=True):
     return pyramidbox(settings, file_list, 'train', shuffle)
 
 
-def test(settings, file_list):
-    return pyramidbox(settings, file_list, 'test', False)
+def val(settings, file_list):
+    return pyramidbox(settings, file_list, 'val', False)
```

Comment:
line 243 'test'->'val', same as this line

Reply:
Done.